src/Compilers/VisualBasic/vbc/vbc.csproj

```diff
@@ -34,25 +34,6 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <PlatformTarget>AnyCPU</PlatformTarget>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
-    <DebugSymbols>true</DebugSymbols>
-    <OutDir>..\..\..\..\Binaries\Debug\amd64</OutDir>
-    <CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
-    <NoWarn>1591</NoWarn>
-    <DebugType>full</DebugType>
-    <PlatformTarget>x64</PlatformTarget>
```

Comment:
Well, implicitly in this pattern is a proper specification of which project targets this project actually targets. i.e. there is no Platform that you can set (after my change) that will result in PlatformTarget=x64. We could centralize but then we might have other things happening we don't like, I'm not sure.